front/src/modules/pathfinding/components/Itinerary/ModalSuggestedVias.tsx

```diff
@@ -110,7 +116,10 @@ const ModalSuggestedVias = ({ suggestedVias }: ModalSuggestedViasProps) => {
           <button
             className="btn btn-danger btn-sm btn-block mb-1"
             type="button"
-            onClick={() => dispatch(clearVias())}
+            onClick={() => {
+              const newPathSteps = [pathSteps.at(0) || null, pathSteps.at(-1) || null];
```

Comment:
This modal can't exist if we miss origin or destination (l.114 in `Itinerary.tsx`) so we should be able to do this
```suggestion
              const newPathSteps = [pathSteps.at(0)!, pathSteps.at(-1)!;
```

Reply:
Done